app/services/publisher_notifier.rb

```diff
@@ -3,22 +3,22 @@ class PublisherNotifier < BaseService
   attr_reader :notification_params, :notification_type, :publisher
 
   # Should match methods in NotificationMailer starting with #publisher_*
-  NOTIFICATION_TYPES = %w(form_retry payments_activated)
+  NOTIFICATION_TYPES = %w(verified_no_wallet)
 
   def initialize(notification_params: {}, notification_type:, publisher:)
     raise ":publisher invalid" if !publisher.is_a?(Publisher)
     @notification_type = notification_type.to_s
     if !NOTIFICATION_TYPES.include?(@notification_type)
-      raise InvalidNotificationTypeError.new("#{notification_type} is an invalid notification_type")
+      raise InvalidNotificationTypeError.new("'#{notification_type}' is an invalid notification type")
     end
     @notification_params = (notification_params && notification_params.to_hash.symbolize_keys) || {}
     @publisher = publisher
   end
 
   def perform
-    NotificationMailer.public_send(mailer_method, @publisher, notification_params).deliver_later
-    if NotificationMailer.should_send_internal_emails? && NotificationMailer.respond_to?(mailer_method_internal)
-      NotificationMailer.public_send(mailer_method_internal, @publisher, notification_params).deliver_later
+    PublisherMailer.public_send(mailer_method, @publisher, notification_params).deliver_later
+    if PublisherMailer.should_send_internal_emails? && PublisherMailer.respond_to?(mailer_method_internal)
```

Comment:
@kpfefferle and I talked about this a bit: There currently are not any internal emails sent via this API endpoint, perhaps we could at least add a comment here saying this codepath is for future use. That or simply remove it.

Reply:
@mrose17 Do we anticipate that emails sent via the API endpoint will need internal copies sent?

Reply:
given we're going to stop logging emails, i think it makes sense now to send the internal redacted copy with any publisher emails.

Reply:
I just added an internal copy method for this initial use case, so this code is now indeed being used.